HomeWork/HomeWork/UserService.cs

```diff
@@ -0,0 +1,578 @@
+        private readonly By _searchingItems = By.XPath("//td[@class='where-buy-description']//h3[text()]");
+
+
+        private const string _registrationName = "User1";
+        private const string _registrationEmail = "danya.sydortsov@tech-stack.io";
+        private const string _registrationPassword = "Password123";
+        private const string _searchingItem = "iPhone 13 Pro 256";
+
+        //public By Computer => By.XPath("//a[@class='cancel-click mainmenu-link']");
+        public By ToCompareTablet => By.XPath("//label[@id='label_2090044']");
+
+        RandomUser rndUs = new RandomUser();
+
+
+        public void Create()
+        {
+            var rndLg = rndUs.CreateRandomLogin();
+
+            var signIn = driver.FindElement(_signInButton);
+            signIn.Click();
+
+            Thread.Sleep(2000);
+
+            var registrationButton = driver.FindElement(_registrationButton);
+            registrationButton.Click();
+
+            Thread.Sleep(1000);
+
+            var inputName = driver.FindElement(_nameInputButton);
+            inputName.SendKeys(rndLg);
+
+            var emailInput = driver.FindElement(_emailInputButton);
+            emailInput.SendKeys(rndUs.CreateRandomEmail());
+
+            var passwordInput = driver.FindElement(_passwordInputButton);
+            passwordInput.SendKeys(rndUs.CreateRandomPassword());
+
+            var acceptRegistration = driver.FindElement(_acceptRegistrationButton);
+            acceptRegistration.Click();
+
+            Thread.Sleep(2000);
+
+            var acceptButton = driver.FindElement(_acceptButton);
+            acceptButton.Click();
+
+            var actualLogin = driver.FindElement(_acceptLogin).Text;
+
+            Assert.AreEqual(actualLogin, rndLg, "Wrong!");
+        }
+
+        public void Search()
+        {
+            var searchInput = driver.FindElement(_searchInputButton);
+            searchInput.SendKeys(_searchingItem);
+
+            var searchButton = driver.FindElement(_searchItemButton);
+            searchButton.Click();
+
+            Thread.Sleep(2000);
+
+            var searchingItems = driver.FindElements(_searchingItems)/*Select(element => element.Text).ToList()*/;
+
+            foreach (var searchingItem in searchingItems)
+            {
+                var a = searchingItem.Text;
+
+                Assert.IsTrue(a.Contains(_searchingItem));
+            }
+
+            // var textItems = searchingItems.ElementAt(0).Text;
+
+            // Assert.IsTrue(_searchingItem.Contains(searchingItems), "Wrong!");
+        }
+
+        public void EnterComputer()
+        {
+            Actions action = new Actions(driver);
+
+            var enterComputer = driver.FindElement(_computerPage);
+            action.MoveToElement(enterComputer).Perform();
+
+            Thread.Sleep(3000);
+
+            var dropComputer = driver.FindElement(_dropComputer);
```

Comment:
where are you drop computer? Hope i can help you to find it! maybe that should be "computerFolderDropdown"?